akvo/rsr/migrations/0205_labeled_period_report.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
```

Comment:
This would need to be re-numbered because we already have another migration on master